download-links.html

```diff
@@ -18,23 +18,23 @@ <h2>Source Downloads</h2>
 <h2>Linux Downloads</h2>
 
 <ul>
-<li>32 bit x86: <a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">https://download.cuberite.org/linux-i386/Cuberite.tar.gz</a></li>
-<li>64 bit x86: <a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz</a></li>
+<li>ia32: <a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">https://download.cuberite.org/linux-i386/Cuberite.tar.gz</a></li>
+<li>amd64: <a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz</a></li>
```

Comment:
Please no, our users have no idea what ia32 is! Even I haven't come across this before, I thought it was itanium for a second.

The previous was fine, people know what 32 bit and 64 bit are.

Reply:
what about x86 and x86-64?

Reply:
If it ain't broken, don't fix it.